### PR TITLE
[Redesign] Update traffic light detection operator

### PR DESCRIPTION
**Pull request by objorkman**
* Add traffic light detector to runner script
* Update `traffic_light_det_operator.py` to redesign
Screen Shot 2022-03-15 at 2 06 12 AM

### CODE REVIEW

File: pylot/perception/detection/traffic_light_det_operator.py [hunk 1]
```diff
@@ -64,66 +59,33 @@ def __init__(self, camera_stream: erdos.ReadStream,
...
 
         self._logger.debug('@{}: {} detected traffic lights {}'.format(
-            msg.timestamp, self.config.name, traffic_lights))
+            context.timestamp, self.config.name, traffic_lights))
 
-        traffic_lights_stream.send(
-            TrafficLightsMessage(msg.timestamp, traffic_lights))
-        traffic_lights_stream.send(erdos.WatermarkMessage(msg.timestamp))
+        context.write_stream.send(
+            TrafficLightsMessage(context.timestamp, traffic_lights))
```
Comment (pschafhalter) [inline pylot/perception/detection/traffic_light_det_operator.py:77]:
```suggestion
        context.write_stream.send(
            Message(context.timestamp, traffic_lights))
```

**Review by pschafhalter — CHANGES_REQUESTED** [review]
Could you fix the merge conflicts?

File: pylot/perception/detection/detection_operator.py [hunk 2]
```diff
@@ -38,11 +38,9 @@ def __init__(self, model_path: str, flags):
...
         # Only sets memory growth for flagged GPU
         physical_devices = tf.config.experimental.list_physical_devices('GPU')
-        tf.config.experimental.set_visible_devices(
-            [physical_devices[self._flags.obstacle_detection_gpu_index]],
-            'GPU')
-        tf.config.experimental.set_memory_growth(
-            physical_devices[self._flags.obstacle_detection_gpu_index], True)
+        tf.config.experimental.set_visible_devices([physical_devices[0]],
+                                                   'GPU')
+        tf.config.experimental.set_memory_growth(physical_devices[0], True)
```
Comment (pschafhalter) [inline pylot/perception/detection/detection_operator.py:43]: Why were these changes made?

Comment (objorkman) [reply pylot/perception/detection/detection_operator.py]: Oops, fixed it!

**Review by pschafhalter — APPROVED** [review]
LGTM

**Pull request merged**